terraform/gcp/modules/tuf/tuf.tf

```diff
@@ -129,8 +129,13 @@ resource "google_storage_bucket_iam_member" "public_tuf_preprod_member" {
 }
 
 resource "google_storage_bucket_iam_member" "tuf_sa_preprod_editor" {
```

Comment:
oops this is for preprod: this likely affects nothing in staging. I'll update this

Reply:
Added the same change for both preprod and prod buckets to keep things simple.